pkgs/development/python-modules/mdpo/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, polib
+, pymd4c
+}:
+
+buildPythonPackage rec {
+  pname = "mdpo";
```

Comment:
from the description, this sounds like it's meant to be used an cli utility.

please add the following to `pkgs/top-level/all-packages.nix` to export this as an application:
```
  mdpo = with python3Packages; toPythonApplication mdpo;
```